classes/book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rentals(rental)
+    @rentals.push(rental)
+    rental.book = self
+  end
```

Comment:
- [ ] The code should create a new Rental object with the date and person as its attributes.

Reply:
@dasileker Thank you for the review, I will fix this issue. Is it ok if I redeclare it like this: 

def add_rentals(rental,date)
    @rentals.push(rental.date)
end?

Please let me know if this is the right implementation.

Reply:
@dasileker So I implemented the new code like this: 

def add_rentals(date, person)
    @rentals << Rental.new(date, self)
end

Is this implementation right?

Reply:
@dasileker I updated the code, but if it is wrong kindly, please guide me with an example, that would be really helpful